Bugfix/too many partitions (#132)
fivetran/dbt_netsuite: CHANGELOG.md

# dbt_netsuite v0.14.0

[PR #132](https://github.com/fivetran/dbt_netsuite/pull/132) includes the following updates:

## Breaking Changes (Full refresh required after upgrading)

> - Partitioned models have had the `partition_by` logic adjusted to include a granularity of a month. This change should only impact BigQuery warehouses and was applied to avoid the common `too many partitions` error some users have experienced do to over partitioning by day. Therefore, adjusting the partition to a month granularity will increase the partition windows and allow for more performant querying. This change was applied to the following models:

Reviewer: ```suggestion
- Partitioned models have had the `partition_by`  granularity adjusted from `day` to `month`. This change should only impact BigQuery warehouses and was applied to avoid the common `too many partitions` error users have experienced due to over-partitioning by day. Therefore, adjusting the partition to a monthly granularity will increase the partition windows and allow for more performant querying. 
- This change was applied to the following models:
```

  - `int_netsuite2__tran_with_converted_amounts`
  - `netsuite2__balance_sheet`
  - `netsuite2__income_statement`
  - `netsuite2__transaction_details`

## Under the Hood

> - Consistency tests add for each Netsuite2 end model to be used during integration test validations.

Reviewer: ```suggestion
- Consistency tests added for each Netsuite2 end model in order to be used during integration test validations.
```
